frontend/src/components/Profile/BasicTabs.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+const defaultProps = {
+  children: 'Firt tab'
+};
+
+function TabPanel(props: TabPanelProps & typeof defaultProps) {
+  const { children, value, index } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
```

Comment:
change div to mui component (Box or another wrapper component)

Reply:
This is standard not modified (in this part) MUI component named Basic Tabs https://mui.com/material-ui/react-tabs/ 
![image](https://user-images.githubusercontent.com/37217530/167613902-dbffe366-69a6-4315-8d38-486b7f47eed0.png)

If it necessary I can try modify it